src/combinatorics/simple_graph/subgraph.lean

```diff
@@ -82,6 +82,9 @@ G'.adj_sub h
 /-- A subgraph is called a *spanning subgraph* if it contains all the vertices of `G`. --/
 def is_spanning (G' : subgraph G) : Prop := ∀ (v : V), v ∈ G'.verts
 
+lemma adj_sub_mt (G' : subgraph G) {u v : G'.verts} (h : ¬G.adj u v) : ¬G'.adj u v :=
+λ h', h (G'.adj_sub h')
```

Comment:
```suggestion
lemma adj_sub_mt (G' : subgraph G) {u v : G'.verts} : ¬G.adj u v → ¬G'.adj u v :=
mt (adj_sub G')
```
Which makes me question whether this lemma is really needed.

Reply:
it wasn't working particularly well for me inline, but seems I was just having a bad day at the office

Reply:
For some reason `mt (G'.adj_sub)` doesn't work, but I think that's a quirk in dot notation which is often responsible for weird elaboration problems.

Reply:
this is just what I was about to say; I think this is why I originally made it this way. I wish dot notation worked properly all of the time :/

Reply:
I'm happy with either, maybe we should make a note of this quirk with dot notation though

Reply:
I'm happy with either, maybe we should make a note of this quirk with dot notation though

Reply:
I think lets just remove it. We're in dire need of some documentation on dot notation, but I don't think it belongs in this file.

Reply:
Yes, I agree it shouldn't be in this file.